pkg/cli/image/manifest/manifest.go

```diff
@@ -558,155 +521,21 @@ func PutManifestInCompatibleSchema(
 	} else {
 		klog.V(5).Infof("Put manifest %s", ref)
 	}
-	switch t := srcManifest.(type) {
-	case *schema1.SignedManifest:
```

Comment:
We have to ensure that these types are not used anywhere, otherwise that would break scripts. We'd need to follow deprecation policy. Notify that we deprecate this in this release and remove in next release after that. Or we have already deprecate it?

Reply:
It looks like this is the link https://access.redhat.com/solutions/4346011 given in PR description.

Reply:
Yes, and it was deprecated by docker back in 2015